### DISCUSSION TITLE
Disable generate_chat_tags logging (or respect logging level settings)

### DISCUSSION THREAD

**Discussion by bgeneto** [open-webui/open-webui#6667] [Issues]
# Bug Report

## Installation Method

Docker compose

## Environment

- **Open WebUI Version:** v0.3.35
- **Ollama (if applicable):** n/a

- **Operating System:** Debian 12
- **Browser (if applicable):** Chrome 130.0

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.
- [x] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

Nothing should be printed in the logs if logging level settings (`GLOBAL_LOG_LEVEL`, e.g.) is set to ERROR or CRITICAL.

## Actual Behavior:

generate_chat_tags logs every request like this:

```

content ### Task:
Generate 1-3 broad tags categorizing the main themes of the chat history, along with 1-3 more specific subtopic tags.

### Guidelines:
- Start with high-level domains (e.g. Science, Technology, Philosophy, Arts, Politics, Business, Health, Sports, Entertainment, Education)
- Consider including relevant subfields/subdomains if they are strongly represented throughout the conversation
- If content is too short (less than 3 messages) or too diverse, use only ["General"]
- Use the chat's primary language; default to English if multilingual
- Prioritize accuracy over specificity

### Output:
JSON format: { "tags": ["tag1", "tag2", "tag3"] }

### Chat History:
<chat_history>
USER: hi
ASSISTANT: Hello! How can I help you today?
</chat_history>

```

## Description

**Bug Summary:**
generate_chat_tags logs every request like this:

## Reproduction Details

**Steps to Reproduce:**
just chat with any model and check your logs with `docker logs open-webui` 

## Logs and Screenshots

**Browser Console Logs:**
n/a

**Docker Container Logs:**
```

content ### Task:
Generate 1-3 broad tags categorizing the main themes of the chat history, along with 1-3 more specific subtopic tags.

### Guidelines:
- Start with high-level domains (e.g. Science, Technology, Philosophy, Arts, Politics, Business, Health, Sports, Entertainment, Education)
- Consider including relevant subfields/subdomains if they are strongly represented throughout the conversation
- If content is too short (less than 3 messages) or too diverse, use only ["General"]
- Use the chat's primary language; default to English if multilingual
- Prioritize accuracy over specificity

### Output:
JSON format: { "tags": ["tag1", "tag2", "tag3"] }

### Chat History:
<chat_history>
USER: hi
ASSISTANT: Hello! How can I help you today?
</chat_history>

```

**Screenshots/Screen Recordings (if applicable):**
n/a

**Comment by tjbck** [comment]
https://github.com/open-webui/open-webui/blob/7228b39064ac28e1240bf8998f2a35535c6f7ef5/backend/open_webui/main.py#L1649

It's already set to `log.debug` here so unsure why it wouldn't comply with `GLOBAL_LOG_LEVEL`. If you manage to find a fix, PR welcome!

**Comment by tjbck** [comment]
Fixed on dev!

**Reply by bgeneto** [reply]
Thanks, because logging was huge with embedding and image gen models!

**Comment by git2212** [comment]
This bug or feature is back... how do we get rid of it?... someone needs to stop talking to the bright idea fairy!
